frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/elements/IGraphElement.java

```diff
@@ -14,9 +12,11 @@
   int getUpElementIndex();
 
   /**
-   * @return Indexes of elements below (in table) that are connected directly in graph to this one.
+   * @return Index of element below (in table) that is connected directly in graph to this one and that is not its subbranch.
```

Comment:
Hmm so am I right that:
* for a commit element, it's either another commit element or the containing branch (never null),
* for a branch element, it's either first contained commit element, or null (never another branch)?

Would be good to provide a breakdown like the above + the reason why we need to the exception for subbranches

Reply:
Probably the word "directly" is unfortunate here. 

You are right about the first "dot". 
The second:
- for a branch element, it's either next sibling branch element or null (if none left)

I will provide updated javadoc